hooks/post_push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# Parse image name for repo name
+tagStart=$(expr index "$IMAGE_NAME" :)  
+repoName=${IMAGE_NAME:0:tagStart-1}
+DATE=`date +%Y-%m-%d`
+# Grabbing the commit tag for master
+git clone https://github.com/broadinstitute/gatk.git tmp
+cd tmp
+NAME=$(git describe)
```

Comment:
Seems like it's not guaranteed that the version you check out will be the same as the version used to build the docker image, which is a problem.